rmw_cyclonedds_cpp/src/rmw_node.cpp

```diff
@@ -1474,6 +1474,9 @@ extern "C" rmw_ret_t rmw_deserialize(
   } catch (rmw_cyclonedds_cpp::Exception & e) {
     RMW_SET_ERROR_MSG_WITH_FORMAT_STRING("rmw_serialize: %s", e.what());
     ok = false;
+  } catch (std::runtime_error & e) {
```

Comment:
Nit:

```suggestion
  } catch (const std::runtime_error & e) {
```

Reply:
I think I'll pick consistency. It'd be good to flesh out this details though (along with error handling normalization). FYI @eboasson.